docker-compose.yml

```diff
@@ -3,6 +3,8 @@ services:
   db:
     image: postgres
     restart: on-failure
+    environment:
+        - POSTGRES_PASSWORD=postgres
```

Comment:
I'm not sure why it's worked this whole time without this. The docs do say it's mandatory. Does it not work for you?

Reply:
This seems to have only changed recently. If the DB was initialised before that it probably is not needed, but bootstrapping fresh, did also require the password for me. https://github.com/docker-library/postgres/commit/f1bc8782e7e57cc403d0b32c0e24599535859f76

Reply:
Don't we also need to add this env variable as `'POSTGRESQL_ADMIN_PASSWORD'` to the services which require the DB, like `rails`, `sidekiq`, etc.?